src/quickgui/qgsquickpositionkit.h

```diff
@@ -114,6 +114,11 @@ class QUICK_EXPORT QgsQuickPositionKit : public QObject
      */
     Q_PROPERTY( QVector<double> simulatePositionLongLatRad READ simulatePositionLongLatRad WRITE setSimulatePositionLongLatRad NOTIFY simulatePositionLongLatRadChanged )
 
+    /**
```

Comment:
Can you add "Internal source of GPS location data." before the existing documentation text?